### PR TITLE
Update testingRococo.ts

### PR DESCRIPTION
**Pull request by icodezjb**
ChainX ParachainId update to 1059
Zenlink ParachainId update to 1188

### CODE REVIEW

File: packages/apps-config/src/endpoints/testingRococo.ts [hunk 1]
```diff
@@ -139,7 +139,7 @@ export function createRococo (t: TFunction): LinkOption[] {
         // Needs paraId >= 1000, below that is reserved for system parachains
         // Update paraId to new selection, remove this flag to enable
         isDisabled: true,
-        paraId: 59,
+        paraId: 1059,
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRococo.ts:142]: Can you just drop the lines above the paraId as well - the isDisabled (and comment), with it included it won't show.

File: packages/apps-config/src/endpoints/testingRococo.ts [hunk 2]
```diff
@@ -369,7 +369,7 @@ export function createRococo (t: TFunction): LinkOption[] {
         // Needs paraId >= 1000, below that is reserved for system parachains
         // Update paraId to new selection, remove this flag to enable
         isDisabled: true,
-        paraId: 188,
+        paraId: 1188,
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRococo.ts:372]: Same here, the 3 lines above can be removed.

File: packages/apps-config/src/endpoints/testingRococo.ts (hunk 1 above)
Comment (icodezjb) [reply packages/apps-config/src/endpoints/testingRococo.ts:139]: Done

File: packages/apps-config/src/endpoints/testingRococo.ts (hunk 2 above)
Comment (icodezjb) [reply packages/apps-config/src/endpoints/testingRococo.ts:366]: Done

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**